### PR TITLE
Fix curl arguments and HTTP proxy examples

### PR DESCRIPTION
**Pull request by maximiliankolb**
#### What changes are you introducing?

* Fix curl arguments: some were duplicate (both "-k" and "--insecure" which is the same)
* Somewhat unfiy the HTTP proxy example names and URLs; always say "http-proxy", not "proxy" to clearly differentiate betwen Smart Proxies and HTTP proxies.

Please review commit by commit.

#### Why are you introducing these changes? (Explanation, links to references, issues, etc.)

Better readability

#### Anything else to add? (Considerations, potential downsides, alternative solutions you have explored, etc.)

I have verified all curl arguments via man page.

#### Checklists

* [ ] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.12/Katello 4.14 (Satellite 6.16)
* [x] Foreman 3.11/Katello 4.13
* [ ] Foreman 3.10/Katello 4.12
* [ ] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8/6.9/6.10)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* We do not accept PRs for Foreman older than 3.5.

### CODE REVIEW

**Review by apinnick — APPROVED** [review]
Mostly suggestions, so I am approving the PR. 

Please fix the code block with backslash on the last line. That's the only change that is mandatory.

File: guides/common/modules/proc_adding-a-default-http-proxy.adoc [hunk 1]
```diff
@@ -22,29 +21,29 @@ To use the CLI instead of the {ProjectWebUI}, see the xref:cli-adding-a-default-
 
 [id="cli-adding-a-default-http-proxy_{context}"]
 .CLI procedure
-
-. Verify that the `http_proxy`, `https_proxy`, and `no_proxy` variables are not set.
+. Verify that the `http_proxy`, `https_proxy`, and `no_proxy` variables are not set:
 +
 [options="nowrap"]
 ----
 # unset http_proxy
```
[the remark this answers is not in the document]
Comment (ekohl) [reply guides/common/modules/proc_adding-a-default-http-proxy.adoc:28]: The bash man page says you can unset multiple so you can use
```console
# unset http_proxy https_proxy no_proxy
```

[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/common/modules/proc_adding-a-default-http-proxy.adoc:28]: Thanks, applied.

File: guides/common/modules/proc_adding-a-default-http-proxy.adoc [hunk 2]
```diff
@@ -22,29 +21,29 @@ To use the CLI instead of the {ProjectWebUI}, see the xref:cli-adding-a-default-
...
 ----
-# hammer http-proxy create --name=_myproxy_ \
---url http://_myproxy.example.com_:8080  \
---username=_proxy_username_ \
---password=_proxy_password_
+# hammer http-proxy create \
+--name=_My_HTTP_Proxy_ \
+--username=_My_HTTP_Proxy_User_Name_
+--password=_My_HTTP_Proxy_Password_ \
+--url http://_http-proxy.example.com_:8080  \
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/common/modules/proc_adding-a-default-http-proxy.adoc:40]: Thanks; fixed line 40 and 38.

File: guides/common/modules/proc_configuring-pxe-files-proxy.adoc [hunk 3]
```diff
@@ -5,30 +5,24 @@ For Red Hat content served through the Content Delivery Network, {SmartProxy} do
 However, when configuring and installing an operating system using Installation Media, {SmartProxy} connects directly using the `wget` utility.
 
 .Procedure
-
-. On {SmartProxy} with the TFTP feature, to verify the ports that are permitted by SELinux for the HTTP cache, enter the following command:
+. On your TFTP {SmartProxy}, to verify the ports that are permitted by SELinux for the HTTP cache, enter the following command:
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/common/modules/proc_configuring-pxe-files-proxy.adoc:8]: Applied.

**Review by maximiliankolb — COMMENTED** [review]
Rebased to "master" and applied all feedback; will merge with green GHA.

**Comment by maximiliankolb** [comment]
Merged to "master" and cherry-picked:
   b9ddb14a2c..2378756ccf  3.12 -> 3.12
   b88843390a..5161c43c2f  3.11 -> 3.11

**Pull request merged**